scripts/post-install.sh

```
@@ -219,6 +238,37 @@ function update_service_templates() {
     log [INFO] "Service template versions updated successfully"
 }
 
+function update_flavor_in_yaml() {
+  if [ ! -f "${GENERATED_POST_INSTALL_DIR}/dpudeployment.yaml" ]; then
+    log "ERROR" "File not found: ${GENERATED_POST_INSTALL_DIR}/dpudeployment.yaml"
+    return 1
+  fi
+
+  local flavor="flavor-$NODES_MTU" 
+
+  log "INFO" "Updating flavor in to $new_flavor"
```

Shellcheck (0.11.0)

[warning] new_flavor is referenced but not assigned.

(SC2154)